### ISSUE TITLE
Back Office / Hardware

### ISSUE THREAD

**Issue by renki81** [allegro/ralph#2328]
Hi
Since the last update i can there are some basic info display error.
Here, the model is no longer displayed. Also the user no longer appears.

I see also this error everytime when I load a page:
/static/bower_components/iron-a11y-announcer/iron-a11y-announcer.html HTTP/1.1" 404 1796

**Comment by renki81** [comment]
Also if you want to assign a new hardware then no model and user fields are displayed.

**Comment by ghost** [comment]
Could you run `gulp` from ralph dir?
You should have `./iron-a11y-announcer` dir. in `bower components` dir.

**Comment by derbalazs** [comment]
Hi Same issue here.
![virtualbox_ralph3_30_03_2016_11_34_18](https://cloud.githubusercontent.com/assets/18074151/14138107/a7996dbc-f66b-11e5-98e1-21d0749a6c36.png)

**Comment by derbalazs** [comment]
![virtualbox_ralph3_30_03_2016_11_38_28](https://cloud.githubusercontent.com/assets/18074151/14138207/33eab870-f66c-11e5-8ca5-d3431eaa2fdd.png)
![screen23444](https://cloud.githubusercontent.com/assets/18074151/14138212/3662d3b2-f66c-11e5-81dc-61fba9e71a3d.png)

**Comment by renki81** [comment]
If I'm looking for after I get the:

sudo find -iname iron-a11y*
./opt/ralph/ralph-core/lib/python3.4/site-packages/ralph/admin/static/bower_components/iron-a11y-keys-behavior
./opt/ralph/ralph-core/lib/python3.4/site-packages/ralph/admin/static/bower_components/iron-a11y-keys-behavior/iron-a11y-keys-behavior.html

**Comment by ghost** [comment]
We got the problem, I'm working on solution.

**Comment by derbalazs** [comment]
Thanks xliiv!

**Comment by ghost** [comment]
@derbalazs thanks for bug report :)
Here's the fix:
https://github.com/allegro/ralph/pull/2330

**Comment by renki81** [comment]
Thanks xliiv for fast fix :+1:

**Comment by derbalazs** [comment]
Thanks. Trying to find out how to do the fix, but I am on it... :)

**Comment by mkurek** [comment]
@derbalazs Fetch new code and run `gulp` in ralph dir (optionally `ralph collectstatic`)

**Comment by derbalazs** [comment]
Thanks

**Comment by mkurek** [comment]
It everything working right now? If yes please close the issue.

**Comment by derbalazs** [comment]
renki81! You can close this. I have all sorted from my side.
